src/os/bluestore/BlueStore.h

```diff
@@ -1789,6 +1789,7 @@ class BlueStore : public ObjectStore,
   uint64_t min_alloc_size = 0; ///< minimum allocation unit (power of 2)
   size_t min_alloc_size_order = 0; ///< bits for min_alloc_size
   uint64_t prefer_deferred_size = 0; ///< size threshold for forced deferred writes
+  int deferred_batch_ops = 0; ///< deferred batch size
```

Comment:
In practice I think as long as the value is a word size or less we will still get an atomic update -- just not immediately.  Which should be okay.  I'm worried that using atomics might introduce a read barrier on read?